site/styles/_pages.scss

```diff
@@ -33,6 +33,18 @@
     }
   }
 
+  h2 {
+    font-size: 30px;
+    font-weight: $font-weight-bold;
+    margin-bottom: 0;
+    @include media-breakpoint-up(md) {
+      font-size: 36px;
+    }
+    @include media-breakpoint-up(xl) {
+      font-size: 38px;
+    }
+  }
+
```

Comment:
If I'm reading this right, h2 defaults to larger bigger than h1 unless it's a desktop view, in which case they're the same? And our h3 is the same as h1 (20px) on the mobile view?

Reply:
Yeah I was a bit confused by these as well as I just followed the styleguide settings provided in Zeplin, I'll raise these issues with Bex, didn't realize they were inconsistent throughtout.

![image](https://user-images.githubusercontent.com/3003043/161462811-9a570972-81d3-47cb-b6fe-851078e786fa.png)

Reply:
Reverted CSS changes. Will look at creating another PR for just the design aspect.